playwright-e2e/dsm/component/tables/participant-list-table.ts

```diff
@@ -57,47 +54,13 @@ export class ParticipantListTable extends Table {
     throw new Error('Failed to get Total number of participants in Participant List table');
   }
 
-  /**
-    * Returns the guid of the most recently created playwright participant
-    * @param isRGPStudy mark as true or false if this is being ran in RGP - parameter is only needed if method is ran in RGP study
-    * @returns the guid of the most recently registered playwright participant
-  */
-  public async getGuidOfMostRecentAutomatedParticipant(participantName: string, isRGPStudy?: boolean): Promise<string> {
```

Comment:
This func, `getGuidOfMostRecentAutomatedParticipant` creates a loop with `ParticipantListPage`:
`ParticipantListPage` -> `ParticipantListTable` -> `ParticipantListPage`.

I moved this func to `ParticipantListPage`.